### DISCUSSION TITLE
Determine how many distinct meshes are return by boolean difference

### DISCUSSION THREAD

**Discussion by eparrott** [libigl/libigl#1822] [Q&A] [answered]
Is there a way to determine if a mesh boolean difference operation between two meshes results in more than one distinct mesh? I.e. in the case of (mesh a)-(mesh b)=(mesh c), can you tell if mesh b completely bisects mesh a such that mesh c is technically composed of two unique watertight sub-meshes? If so, can you then tell which vertices of mesh c are contained in each distinct sub-mesh?

**Comment by alecjacobson** [comment, marked as answer]
Do you mean connected components? If so you could use igl::connected_components

**Reply by eparrott** [reply]
Thank you! That's exactly what I was looking for!
